plugins/python-build/bin/pyenv-install

```diff
@@ -271,7 +271,7 @@ for DEFINITION in "${DEFINITIONS[@]}"; do
         echo "  brew update && brew upgrade pyenv"
       elif [ -d "${here}/.git" ]; then
         printf ":\n\n"
-        echo "  cd ${here} && git pull && cd -"
+        echo "  git -C \"$(realpath "${here}")\" pull"
```

Comment:
Why the need for `realpath`?

Reply:
We don't even load our `realpath` fallback here (it's not preinstalled in e.g. OSX).

Reply:
`realpath` is just to make the path look nicer, without `..` and stuff.
I can get rid of it :)